content/doc/book/security/securing-jenkins.adoc

```diff
@@ -12,30 +13,7 @@ ifndef::env-github[:imagesdir: ../../resources]
 :toc: left
 endif::[]
 
-= Securing Jenkins
-
-Securing Jenkins has two aspects to it.
-
-* Access control, which ensures users are authenticated when accessing Jenkins
-  and their activities are authorized.
-* Protecting Jenkins against external threats
-
-== Access Control
-
-You should lock down the access to Jenkins UI so that users are authenticated
-and appropriate set of permissions are given to them. This setting is
-controlled mainly by two axes:
-
-* *Security Realm*, which determines users and their passwords, as well as what
-  groups the users belong to.
-* *Authorization Strategy*, which determines who has access to what.
-
-These two axes are orthogonal, and need to be individually configured. For
-example, you might choose to use external LDAP or Active Directory as the
-security realm, and you might choose "everyone full access once logged in" mode
-for authorization strategy. Or you might choose to let Jenkins run its own user
-database, and perform access control based on the permission/user matrix.
-
+This page is a placeholder for links to some old Wiki pages until we can properly dispose of this information.
 
 * https://wiki.jenkins.io/display/JENKINS/Quick+and+Simple+Security[Quick and Simple Security] --- if you are running Jenkins like `java -jar jenkins.war` and only need a very simple setup
```

Comment:
Just redirects to https://www.jenkins.io/doc/book/installing/war-file/ now.

https://github.com/jenkins-infra/docker-confluence-data/blob/main/content/JENKINS/Quick-and-Simple-Security.html is pretty terrible, so we can just delete this reference.

It discusses the legacy container security realm (with a single admin user, same as setup wizard, just worse) and how to use the (deprecated) "build token" functionality.